docs/src/advanced/assembly.md

```diff
@@ -23,6 +23,27 @@ pub fn add_1(num: u32) -> u32 {
 }
 ```
 
+An `asm` block can only return a single register. If you really need to return more than one value, you can return a tuple. Here's an example showing how you might return a tuple of `(u64, u64)`:
+
+```sway
+pub fn adder(a: u64, b: u64, c: u64) -> (u64, u64) {
```

Comment:
Put this in an actual Sway file in `examples/` so it can be tested as part of CI.